demo_python_at/commons/message.py

```diff
@@ -4,42 +4,91 @@
 class Message(ABC):
     @abstractmethod
     def data(self) -> str:
+        """Abstract method to return message data in string format."""
         pass
 
     @abstractmethod
     def exist(self) -> bool:
+        """Abstract method to verify if message exist and return bool."""
         pass
 
 
 class StrMessage(Message):
     def __init__(self, message: str):
+        """
+        Initialize StrMessage object.
+
+        :param message: data in string format
+        """
         self._message = message
 
     def exist(self) -> bool:
+        """
+        Verify that message length greater than 0.
+
+        :return: True if message length greater than 0
```

Comment:
@AntonKarabaza Please correct grammar mistakes. Its right to say "`is` greater than".